atlas/scripts/utils/fasta.py

```diff
@@ -54,7 +54,7 @@ def split(fasta_file,maxSubsetSize,out_dir, simplify_headers=True):
     SubsetSize= int(ceil( N/ceil(N/ maxSubsetSize)))
     extension= os.path.splitext(fasta_file)[-1]
 
-    os.makedirs(out_dir)
+    os.makedirs(out_dir, exist_ok=True)
```

Comment:
Could you add a check that the file `subset_1.fast` (I'm not shure of the exact spelling)) doesn't exist in this folder. 

I want to escape the case that it halve overwrites these files.

Reply:
@SilasK Sure, I can add this check. Will get back to you once done.